ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -19,7 +19,8 @@ import { inject as service } from '@ember/service';
  * ```
  */
 
-const QUICK_ACTION_ENGINES = ['pki', 'kv', 'database'];
+const QUICK_ACTION_ENGINES = ['pki', 'database'];
```

Comment:
@hellobontempo I decided to just refactor the quick actions cards in this PR since it hasn't been reviewed. Let me know if you have any thoughts/feedback on it!